CHANGELOG.md

```diff
@@ -52,6 +52,127 @@ pay attention to. Medium impact changes are also worth looking at.
 
 <!--releases-->
 
+## 2024-10-01: 2024q3 release
+
+This release covers everything from 2024-07-01 and has been tested with avr-gcc
+v12.2.0 from Upstream and arm-none-eabi-gcc v12.2.1 from xpack.
+
+Breaking changes:
+
+- Deprecate `modm::allocator::Dynamic` in favor of `std::allocator`.
+
+Features:
+
+- Add empty polyfill for fibers when module is not selected.
+- Replace modm containers implementation with libstdc++.
+- Add support for a custom allocator in the `modm:ui` module.
+- Add IWDG driver for STM32H7.
+- Add FDCAN driver for STM32H7.
+- Support for STM32C0 devices.
+
+Integrated Projects:
+
+- ETL upgraded to v20.38.17.
+- STM32F3 headers upgraded to v2.3.8.
+- STM32F7 headers upgraded to v1.2.9.
+- STM32F4 headers upgraded to v2.6.10.
+- STM32G4 headers upgraded to v1.2.4.
+- STM32C0 headers upgraded to v1.2.0.
+- STM32U0 headers added at v1.1.0.
+
+Fixes:
+
+- Fix `modm::GraphicDisplay::clearPixel()` implementation.
+- Fix wrong ADC channel selection on STM32C0/G0/F0.
+- Fix fiber poll timeout when yield takes too long.
+- Fix infinite loop in fiber poll when duration is 0.
+- Ensure fiber poll minimum duration is 1µs/1ms.
+- Add missing device definitions to CMSIS-DSP headers.
+
```

Comment:
[[stm32] Fix initialization of FDCAN message RAM](https://github.com/modm-io/modm/commit/3614dc0b95ecf6beaab6d9da6b4120cdbf977073)